projects/VitalRecord/VitalRecord_util.cs

```diff
@@ -129,7 +118,7 @@ protected void UpdateEntry(bool shouldExist, [CallerMemberName] string propertyN
         /// <param name="section">the section of the composition the Observation should be added to</param>
         /// <param name="extensionURL">if present, specifies that the value should be set on an extension with the provided URL instead</param>
         /// <param name="propertyName">the name of the C# property, used to determine the subject ID</param>
-        protected void SetObservationValue(Dictionary<string, string> value, string code, string codeSystem, string text, string profileURL, string section, string extensionURL = null, [CallerMemberName] string propertyName = null)
+        protected Observation CreateObservation(string code, string codeSystem, string text, string profileURL, string section, string focusId = null, [CallerMemberName] string propertyName = null)
```

Comment:
Should add focusId to the param list above.

Reply:
added